consumers/aws.go

```diff
@@ -53,6 +53,7 @@ func withClient(c AWSClient, groupID string) *awsConsumer {
 func (a *awsConsumer) Sync(endpoints []*pkg.Endpoint) error {
 	kubeRecords, err := a.endpointsToRecords(endpoints)
 	if err != nil {
+		log.Errorf("failed to convert endpoints to RRS: %v. Aborting sync...", err)
 		return err
```

Comment:
yup, there are few other places where it needs to be changed, but probably will do it in next PR